app/views/shared/footer/_footer_newsletter.html.erb

```diff
@@ -3,11 +3,12 @@
     <h2 class="footer__box-title">Stay in touch</h2>
     <p>Subscribe to our newsletter</p>
     <form>
-      <a 
-        href="https://forms.office.com/Pages/ResponsePage.aspx?id=NBUaP4MKZUy1wr4a_vFI5d0WGIwI8X5OtKDnEfbzjC9UQ0NXTEZOMDAxTFZMMlJNVUpWN0NLU0o3Ry4u" 
-        target="_blank"
-        class="btn btn--primary btn--full-width"
-      >Subscribe</a>
+      <%= link_to(
+        'Subscribe',
+        t('fincap.links_external.newsletter'),
+        target: '_blank',
+        class: 'btn btn--primary btn--full-width'
+      ) %>
     </form>
```

Comment:
I know that is not related to your PR, but do you know why there is a form tag around the link?

Reply:
@tomas-stefano good one, no clue. Apparently removing it doesn't impact the button style or the clickability, so I've removed it.

Reply:
Cool! Approved!